### DISCUSSION TITLE
Implement logr LogSink interface

### DISCUSSION THREAD

**Discussion by secustor** [go-kit/log#17] [Ideas]
We are using modules from different projects where each is using a separate logging library.     
To solve this we are using Logr which provides an interface which has been already implemented in many [different go logging solutions](https://github.com/go-logr/logr#implementations-non-exhaustive).     

I have seen https://github.com/go-kit/log/discussions/5, but does this apply here too?     

This would definitely ease the adoption of go-kit.

**Comment by ChrisHines** [comment]
I think the answer is yes. `go-kit/log` defines a simple interface that anyone can use or implement without need to add code to this repo. It looks like many of the Logr adapters are hosted in the same Github org as the core package. I recommend doing the same for a `go-kit/log` adapter for Logr if it is needed. I don't think that the go-kit project wants to maintain it.

**Comment by tonglil** [comment]
Hello @secustor, I've written an adapter for logr recently: https://github.com/tonglil/gokitlogr

**Reply by secustor** [reply]
Thanks! I will definitely take look!
